### ISSUE TITLE
Fix failing GenreStatistics tests

### ISSUE THREAD

**Issue by knjk04** [Project-Books/book-project#673]
**Describe the bug**

- `notShowMostLikedGenreWhenOnlyOneReadGenreExists()`
- `notShowLeastLikedGenreWhenOnlyOneReadGenreExists()`

The above tests fail. There are more books in the read shelf than is expected. As we rely on StatisticTestUtils, these tests can be hard to follow, so they need to be rewritten to make it easier to reason about

**Comment by tmrd993** [comment]
Is this fixed? The tests run fine for me (W10).

Also, this is probably not related but why are we using the actual service beans for the genre statistics tests? Shouldn't we create mock services for this? That way, we probably don't have to reset the bookservice for the tests. They seem to run longer than typical tests probably because they require database connections? Not too sure about this, maybe you can clarify.

**Comment by knjk04** [comment]
No, they fail for me on Windows 10. I've disabled them for the time being because they need to be rewritten.

I agree that we should be mocking the services for this. It will be a lot quicker, like you said
